src/frontend/web_application/src/components/DeviceForm/presenter.jsx

```diff
@@ -0,0 +1,201 @@
+import { FormGrid, FormRow, FormColumn, Fieldset, Legend, TextFieldGroup, SelectFieldGroup } from '../../components/form';
+
+import './style.scss';
+
+
+const DisplayDevice = ({ device, deviceTypes, deviceIPs, insertDate, lastSeenDate, __ }) => (
+  <div>
+    <Section className="m-device__pi">
+      <PiBar level={device.pi ? device.pi : 0} />
+    </Section>
+
+    <Section
+      title={__('device.manage.title')}
+      descr={__('device.manage.descr')}
+    >
+      <FormGrid className="m-device__form">
+        <Fieldset className="m-device__fieldset">
+          <Legend>{__('device.manage_form.name.label')}</Legend>
+          <FormRow reverse>
+            <FormColumn bottomSpace size="medium" className="m-device__infotext">
+              <label htmlFor="device-name">{__('device.manage_form.name.infotext')}</label>
+            </FormColumn>
+            <FormColumn bottomSpace size="medium">
+              <TextFieldGroup
+                label={__('device.manage_form.name.label')}
+                name="device-name"
+                id="device-name"
+                showLabelforSr
+                key={device.name}
+                defaultValue={device.name}
+              />
+            </FormColumn>
+          </FormRow>
+        </Fieldset>
+        <Fieldset className="m-device__fieldset">
+          <Legend>{__('device.manage_form.type.label')}</Legend>
+          <FormRow reverse>
+            <FormColumn bottomSpace size="medium" className="m-device__infotext">
+              <label htmlFor="device-type">{__('device.manage_form.type.infotext')}</label>
+            </FormColumn>
+            <FormColumn size="medium">
+              <SelectFieldGroup
+                className="m-device__type"
+                label={__('device.manage_form.type.label')}
+                name="device-type"
+                id="device-type"
+                showLabelforSr
+                value={device.type}
+                options={deviceTypes}
+              />
+            </FormColumn>
+          </FormRow>
+        </Fieldset>
+        <Fieldset className="m-device__fieldset">
+          <Legend>{__('device.manage_form.ips.label')}</Legend>
+          <FormRow reverse>
+            <FormColumn bottomSpace className="m-device__infotext" size="medium">
+              <label htmlFor="device-ips">{__('device.manage_form.ips.infotext')}</label>
+            </FormColumn>
+            <FormColumn bottomSpace size="medium">
+              <div className="m-device__ip">
+                <TextFieldGroup
+                  label={__('device.manage_form.add-ip.label')}
+                  placeholder={__('device.manage_form.add-ip.label')}
+                  name="device-ips"
+                  defaultValue=""
+                  className="m-device__ip-input"
+                  showLabelforSr
+                />
+                <Button plain inline className="m-device__ip-button"><Icon type="plus" /></Button>
```

Comment:
it would be nice for accessibility to add a title property to the button or a span with the show for sr classname with the purpose of the button, E.G: 

```
<Button ...><Icon ... /><span className="show-for-sr">{__('device.manage_form.action.add-ip')}</span></Button>
```